### PR TITLE
HRIS-104-01 [Markup] Update File for Overtime Modal

### PR DESCRIPTION
**Pull request by jsvelte**
## Issue Link

https://framgiaph.backlog.com/view/HRIS-104

## Definition of Done

- [x] Updated the File for Overtime Modal based on the meeting plan
- [x] Added new actions for Overtime in the table 

## Notes
- This is revision of #104 

## Pre-condition
cd `client` && `npm run dev`
cd `api` && `dotnet run watch`

## Expected Output

- It should now have an updated modal for filing Overtime 

## Screenshots/Recordings
[Sun_ HRIS - My Daily Time Record.webm](https://user-images.githubusercontent.com/108642414/222348066-22e57e5f-5774-4bdf-a04d-5055e7ffc68b.webm)

### CODE REVIEW

File: client/src/components/molecules/OvertimeManagementTable/UpdateOvertimeModal.tsx [hunk 1]
```diff
@@ -48,11 +93,21 @@ const UpdateOvertimeModal: FC<Props> = ({ isOpen, closeModal, row }): JSX.Elemen
...
+    name: 'projects'
+  })
+
   // This will handle Submit and Save New Overtime
   const handleSave = async (data: NewOvertimeFormValues): Promise<void> => {
     return await new Promise((resolve) => {
       setTimeout(() => {
-        alert(JSON.stringify({ ...data, projects: { ...selectedOtherProjectOption } }, null, 2))
+        // console.log(data)
+        alert(JSON.stringify(data, null, 2))
```
Comment (abduljalilpalala) [inline client/src/components/molecules/OvertimeManagementTable/UpdateOvertimeModal.tsx:110]: Remove comment if not needed

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]
I left a minor comment 💬

**Review by arden-dave — APPROVED** [review]
Approved. 👍

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**